test/controllers/notices-setup.controller.test.js

```diff
@@ -809,6 +810,39 @@ describe('Notices Setup controller', () => {
     })
   })
 
+  describe('notices/setup/{sessionId}/preview-return-forms', () => {
+    describe('GET', () => {
+      let buffer
+      beforeEach(async () => {
```

Comment:
I don't think this needs to be asyncronous. Same for a lot of the other `beforeEach` in this test file.
```suggestion
      beforeEach(() => {
```